pkgs/by-name/de/delineate/package.nix

```diff
@@ -0,0 +1,93 @@
+{
+  appstream,
+  buildNpmPackage,
+  cargo,
+  cmake,
+  desktop-file-utils,
+  fetchFromGitHub,
+  gtk4,
+  gtksourceview5,
+  lib,
+  libadwaita,
+  meson,
+  ninja,
+  nix-update-script,
+  pkg-config,
+  rustPlatform,
+  rustc,
+  stdenv,
+  webkitgtk_6_0,
+  wrapGAppsHook4,
+}:
+let
+  d3-graphviz = buildNpmPackage rec {
+    pname = "d3-graphviz";
+    version = "5.6.0";
+
+    src = fetchFromGitHub {
+      owner = "magjac";
+      repo = "d3-graphviz";
+      tag = "v${version}";
+      hash = "sha256-MZhAzR6+GIBTsLPJq5NqaEPHjiBMgYBJ0hFbDPNPgFk=";
+    };
+
+    npmDepsHash = "sha256-J1kptumP/8UoiYDM+AJOYUne0OSkMXCTAXW3ZmavU4E=";
+
+    # keep the devDependencies, as Delineate imports d3 via node_modules
+    # https://github.com/SeaDve/Delineate/blob/v0.1.0/data/graph_view/index.html#L10-L11
+    npmPruneFlags = "--include=dev";
+  };
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "delineate";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "SeaDve";
+    repo = "Delineate";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-dFGh7clxc6UxQRTsNKrggWDvL3CPmzJmrvO1jqMVoTg=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit (finalAttrs) src;
+    hash = "sha256-RtQnpbjULtnvlc71L4KIKPES0WRSY2GoaIwt8UvlYOA=";
+  };
+
+  nativeBuildInputs = [
+    desktop-file-utils
+    gtk4
+    meson
+    ninja
+    pkg-config
+    rustPlatform.bindgenHook
+    rustPlatform.cargoSetupHook
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [
+    appstream
+    cargo
+    cmake
```

Comment:
```suggestion
```
that only goes to nativeBuildInputs

Reply:
I misunderstood `nativeBuildInputs` - I thought that merely prevented hooks from running, which I do not want to run for `cmake` - that's why added it here.
I guess I'll add `dontUseCmakeConfigure = true;`